src/views/Login.vue

```diff
@@ -67,16 +67,16 @@ export default {
       this.error = null
 
       try {
-        const result = await axios.get('/api/ping', {
-          headers: { Authorization: `Bearer ${this.token}` }
-        })
+        const result = await axios.post('/admin/login?password=' + this.token)
```

Comment:
```suggestion
        const result = await axios.post('/admin/login`, '', {
          params: { password: this.token }
        })
```

If you don't want to have injection problems, use axios's `params` option.

Reply:
Thanks!